src/main/resources/reference-data/uc-decision-notice-questions.txt

```diff
@@ -0,0 +1,700 @@
+   "DisplayOrder" : "6",
+   "ListElementCode":"makingSelfUnderstood",
+   "ListElement":"6. Making self understood through speaking, writing, typing, or other means which are normally or could reasonably be used, unaided by another person."
+},
+{
+   "LiveFrom":"01/01/2018",
+   "ID":"FL_physicalDisabilities",
+   "DisplayOrder" : "5",
+   "ListElementCode":"manualDexterity",
+   "ListElement":"5. Manual dexterity."
+},
+{
+   "LiveFrom":"01/01/2018",
+   "ID":"FL_physicalDisabilities",
+   "DisplayOrder" : "4",
+   "ListElementCode":"pickingUp",
+   "ListElement":"4. Picking up and moving or transferring by the use of the upper body and arms."
+},
+{
+   "LiveFrom":"01/01/2018",
+   "ID":"FL_physicalDisabilities",
+   "DisplayOrder" : "3",
+   "ListElementCode":"reaching",
+   "ListElement":"3. Reaching."
+},
+{
+   "LiveFrom":"01/01/2018",
+   "ID":"FL_physicalDisabilities",
+   "DisplayOrder" : "2",
+   "ListElementCode":"standingAndSitting",
+   "ListElement":"2. Standing and sitting."
+},
+{
+   "LiveFrom":"01/01/2018",
+   "ID":"FL_physicalDisabilities",
+   "DisplayOrder" : "1",
+   "ListElementCode":"mobilisingUnaided",
+   "ListElement":"1. Mobilising unaided by another person with or without a walking stick, manual wheelchair or other aid if such aid is normally or could reasonably be worn or used."
+},
+{
+   "LiveFrom":"01/01/2018",
+   "ID":"FL_mentalAssessment",
+    "DisplayOrder" : "7",
+   "ListElementCode":"appropriatenessOfBehaviour",
+   "ListElement":"17. Appropriateness of behaviour with other people, due to cognitive impairment or mental disorder."
+},
+{
+   "LiveFrom":"01/01/2018",
+   "ID":"FL_mentalAssessment",
+   "DisplayOrder" : "6",
+   "ListElementCode":"socialEngagement",
+   "ListElement":"16. Coping with social engagement due to cognitive impairment or mental disorder."
+},
+{
+   "LiveFrom":"01/01/2018",
+   "ID":"FL_mentalAssessment",
+   "DisplayOrder" : "5",
+   "ListElementCode":"gettingAbout",
+   "ListElement":"15. Getting about."
+},
+{
+   "LiveFrom":"01/01/2018",
+   "ID":"FL_mentalAssessment",
+   "DisplayOrder" : "4",
+   "ListElementCode":"copingWithChange",
+   "ListElement":"14. Coping with change."
+},
+{
+   "LiveFrom":"01/01/2018",
+   "ID":"FL_mentalAssessment",
+   "DisplayOrder" : "3",
+   "ListElementCode":"personalAction",
+   "ListElement":"13. Initiating and completing personal action (which means planning, organisation, problem solving, prioritising or switching tasks)."
+},
+{
+   "LiveFrom":"01/01/2018",
+   "ID":"FL_mentalAssessment",
+   "DisplayOrder" : "2",
+   "ListElementCode":"awarenessOfHazards",
+   "ListElement":"12. Awareness of everyday hazards (such as boiling water or sharp objects)."
+},
+{
+   "LiveFrom":"01/01/2018",
+   "ID":"FL_mentalAssessment",
+   "DisplayOrder" : "1",
+   "ListElementCode":"learningTasks",
+   "ListElement":"11. Learning tasks."
+},
+{
+   "LiveFrom":"01/01/2018",
+   "ID": "FL_esaWriteFinalDecisionMobilisingUnaidedQuestion",
```

Comment:
should these `esa` ID's be replaced with `uc`?

Reply:
As above, I've now changed to uc